### PR TITLE
Add hexo-theme-keep w/ npm auto-update

### PR DESCRIPTION
**Pull request by XPoet**

### CODE REVIEW

File: packages/h/hexo-theme-keep.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "autoupdate": {
+    "source": "npm",
+    "target": "hexo-theme-keep",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+          "js/**/*",
+          "font/**/*",
+          "images/**/*"
```
Comment (MattIPv4) [inline packages/h/hexo-theme-keep.json:33]: Please avoid using unrestricted globstars, as they can lead to including resources on the CDN that aren't listed. Please restrict the extensions you are matching, and ideally the directories as well.

Comment (XPoet) [reply packages/h/hexo-theme-keep.json]: I have modified it, please help me merge it, thank you very much.

[commit details](https://github.com/cdnjs/packages/pull/1679/commits/9a6dfdb3daa9f83755f3652daff01226f1fee801)

File: packages/h/hexo-theme-keep.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "hexo-theme-keep",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+          "js/**/*.js",
+          "font/**/*.css"
```
Comment (MattIPv4) [inline packages/h/hexo-theme-keep.json:32]:
```suggestion
          "font/css/*.css",
          "font/webfonts/*.@(ttf|woff2)"
```
I suspect you also need the actual font files for the fonts?

Comment (XPoet) [reply packages/h/hexo-theme-keep.json]: > I suspect you also need the actual font files for the fonts?

Yes. Thanks for reminding

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**